work-process.md

```diff
@@ -4,12 +4,22 @@
 Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)), but modified slightly to apply better to our specific needs:
 
  * we don't do customer projects, so we are our own Product Owners,
- * we currently use only GitHub for sprint management,
+ * we currently use GitHub and ZenHub for sprint management,
  * we work asynchronously, so we try to keep scheduled meetings to the bare minimum.
 
-Undefined issues are created in various repositories. Support issues, being reactionary, stay in project-related repositories and are not included in sprints. When undefined issues in various repositories are to be moved into a sprint, Scrum Master creates a User Story and links to the original issue in the Acceptance Criteria. Sprint is managed from the [Operations repository](https://github.com/niteoweb/operations/issues).
+Undefined issues are created in various repositories. Support issues, being reactionary, stay in project-related repositories and are not included in sprints. When undefined issues in various repositories are to be moved into a sprint, User Story needs to be created for that issue with well defined Acceptance Criteria and Definition of Done. This can be done by the author of the issue, Scrum Master or Project Owner. Sprint is managed from the [Operations repository](https://github.com/niteoweb/operations/issues).
```

Comment:
Yes, we have. Are you referring to this: `Sprint is managed from the [Operations repository]`? If this sentence bothers you we can remove it. You can actually manage sprint from every repository.